### PR TITLE
feat(meilisearch): Meili Search engine

### PR DESCRIPTION
**Pull request by jarosevcik**

### CODE REVIEW

File: zoo/meilisearch/migrations/0001_index_existing_models.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    client.create_index(uid="services", options={"name": "Service", "primaryKey": "id"})
+    client.create_index(
+        uid="dependencies", options={"name": "Dependency", "primaryKey": "id"}
+    )
+    client.create_index(uid="open-api", options={"name": "Schema", "primaryKey": "id"})
+
+
+def remove_indexes(apps, schema_editor):
+    client = meilisearch.Client(
+        "http://meilisearch:7700", settings.MEILISEARCH_MASTER_KEY
```
Comment (aexvir) [inline zoo/meilisearch/migrations/0001_index_existing_models.py]: We should have this configurable via env var, and we can default to `"http://meilisearch:7700"` in a single place

```suggestion
        settings.MEILISEARCH_ENDPOINT, settings.MEILISEARCH_MASTER_KEY
```

Comment (jarosevcik) [reply zoo/meilisearch/migrations/0001_index_existing_models.py]: sure, this is still WIP mr

Comment (aexvir) [reply zoo/meilisearch/migrations/0001_index_existing_models.py]: alrighty, converted to draft

**Review by aexvir — APPROVED** [review]
Good job! 🙂️

**Pull request merged**